src/Compilers/CSharp/Test/Syntax/Generated/Syntax.Test.xml.Generated.cs

```diff
@@ -10106,7 +10107,7 @@ private static ExternAliasDirectiveSyntax GenerateExternAliasDirective()
             => SyntaxFactory.ExternAliasDirective(SyntaxFactory.Token(SyntaxKind.ExternKeyword), SyntaxFactory.Token(SyntaxKind.AliasKeyword), SyntaxFactory.Identifier("Identifier"), SyntaxFactory.Token(SyntaxKind.SemicolonToken));
 
         private static UsingDirectiveSyntax GenerateUsingDirective()
-            => SyntaxFactory.UsingDirective(SyntaxFactory.Token(SyntaxKind.UsingKeyword), default(SyntaxToken), default(NameEqualsSyntax), GenerateIdentifierName(), SyntaxFactory.Token(SyntaxKind.SemicolonToken));
+            => SyntaxFactory.UsingDirective(default(SyntaxToken), SyntaxFactory.Token(SyntaxKind.UsingKeyword), default(SyntaxToken), default(NameEqualsSyntax), GenerateIdentifierName(), SyntaxFactory.Token(SyntaxKind.SemicolonToken));
```

Comment:
nit: the old overload is now not tested.

Reply:
> nit: the old overload is now not tested.

I find existing usage of that overload in the solution sufficient to provide coverage.

---
In reply to: [582088799](https://github.com/dotnet/roslyn/pull/51426#discussion_r582088799) [](ancestors = 582088799)